content/installation/java/package/scheduled-updates-curl.md

```diff
@@ -0,0 +1,134 @@
+<!--
+title: "Schedule Java agent Updates with curl"
+description: "How to use cron and curl to schedule Java agent updates from Maven Central"
+tags: "java agent deployment maven central cron curl auto-update update"
+-->
+
+## About Updates with Maven Central 
+
+Some users like to automatically update their Contrast Java agent software to the latest version. Linux users can schedule Java agent updates from Maven Central using common Linux tools `cron` and `curl`. The following instructions show you how to configure a scheduled Java agent job on an Ubuntu 18.04 Linux host.
+
+> **Note:** Use your preferred editor to create a file with the following contents. The examples provided use `tee` to create the file. 
+
+## Before You Start
+
+If you want to perform each step as you follow along with this guide, you can use [Vagrant](https://www.vagrantup.com/) to create a new Ubuntu 18.04 virtual machine: 
```

Comment:
We should also instruct users to download VirtualBox here: https://www.virtualbox.org/wiki/Downloads

Reply:
>>> If you want to perform each step as you follow along with this guide, you can use [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/wiki/Downloads) to create a new Ubuntu 18.04 virtual machine:

Reply:
Good catch, added